lampod/src/actions/handler.rs

```diff
@@ -31,7 +34,7 @@ pub struct LampoHandler {
     wallet_manager: Arc<dyn WalletManager>,
     chain_manager: Arc<LampoChainManager>,
     external_handlers: RefCell<Vec<Arc<dyn ExternalHandler>>>,
-    #[allow(dead_code)]
+    liquidity_manager: Option<Arc<LampoLiquidityManager>>,
```

Comment:
ok you already have it inside the handler, so in lampo testing you do not need it